.github/workflows/pre_merge.yml

```diff
@@ -17,6 +17,6 @@ jobs:
         with:
           python-version: 3.8
       - name: Install dependencies
-        run: python -m pip install tox
+        run: python -m pip install tox pylint
```

Comment:
we could perhaps add `pylint` into `requirements/dev.txt`, and install `dev.txt`. This would ensure that we always install the same thing?